.claude/skills/restructure-commits/SKILL.md

```
@@ -1,11 +1,24 @@
+  WIP commits and creating clean conventional commit history.
 ---
 
 # Restructure Commits by Component
 
-Reorganize all commits on a feature branch into logical, component-based commits that match HyperShift's architecture.
+Reorganize all commits on a feature branch into logical, component-based commits that match
+HyperShift's architecture.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)